cypress/integration/data-import/modify-marcBib-record.spec.js

```diff
@@ -0,0 +1,105 @@
+import fieldMappingProfiles from '../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import getRandomPostfix from '../../support/utils/stringTools';
+import actionProfiles from '../../support/fragments/data_import/action_profiles/actionProfiles';
+import newJobProfile from '../../support/fragments/data_import/job_profiles/newJobProfile';
+import jobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import matchProfiles from '../../support/fragments/data_import/match_profiles/match-profiles';
+import dataImport from '../../support/fragments/data_import/dataImport';
+import topMenu from '../../support/fragments/topMenu';
+import logs from '../../support/fragments/data_import/logs';
+
+describe('ui-data-import: Verify the possibility to modify MARC Bibliographic record', () => {
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+    cy.getToken(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+  });
+
+  afterEach(() => {
+    cy.getRecordApi()
+      .then(({ body }) => {
+        cy.deleteRecordFromStorageApi(body.records[body.records.length - 1].id);
+      });
+  });
+
+  it('C345423 Verify the possibility to modify MARC Bibliographic record', { tags: [testTypes.smoke] }, () => {
+    // unique name for profiles
+    const mappingProfileName = `autoTestMappingProf.${getRandomPostfix()}`;
+    const actionProfileName = `autoTestActionProf.${getRandomPostfix()}`;
+    const matchProfileName = `autoTestMatchProf.${getRandomPostfix()}`;
+    const jobProfileName = `autoTestJobProf.${getRandomPostfix()}`;
+
+    // file name
+    const nameMarcFileForUpload = `autotestFile.${getRandomPostfix()}.mrc`;
+
+    // create Field mapping profile
+    const mappingProfile = {
+      name: mappingProfileName
+    };
+
+    settingsDataImport.goToMappingProfiles();
+    fieldMappingProfiles.createModifyMappingProfile(mappingProfile);
+    fieldMappingProfiles.checkMappingProfilePresented(mappingProfileName);
+
+    // create Action profile and link it to Field mapping profile
+    const actionProfile = {
+      name: actionProfileName,
+      action: 'Modify (MARC record types only)',
+      typeValue: 'MARC Bibliographic',
+    };
+
+    settingsDataImport.goToActionProfiles();
+    actionProfiles.createActionProfile(actionProfile, mappingProfile);
+    actionProfiles.checkActionProfilePresented(actionProfileName);
+
+    // create Match profile
+    const matchProfile = {
+      profileName: matchProfileName,
+      incomingRecordFields: {
+        field: '001',
+      },
+      existingRecordFields: {
+        field: '001',
+      },
+      matchCriterion: 'Exactly matches',
+      existingRecordType: 'MARC_BIBLIOGRAPHIC'
+    };
+
+    settingsDataImport.goToMatchProfiles();
+    matchProfiles.createMatchProfile(matchProfile);
```

Comment:
steps 12 and 16 related with 2 match profiles creation. You test has jut one step with match profile creation. Please check your design/sync testrail and the cypress code

Reply:
It must be job profile in step 16. I fixed it in the test case.